services.php

```diff
@@ -77,9 +78,20 @@
         'eddbk_screen_options_cache' => function ($c) {
             return new MemoryMemoizer();
         },
+
+        /**
+         * WP Rest Api nonce.
+         *
+         * @since [*next-version*]
+         *
+         * @return NonceInterface
+         */
         'eddbk_wp_rest_nonce' => function (ContainerInterface $c) {
-            return new WpNonce('wp_rest');
+            return $c->get('eddbk_rest_api_nonce_factory')->make([
+                NonceFactoryInterface::K_CONFIG_ID => 'wp_rest'
```

Comment:
Move the value `wp_rest` to config, under maybe `wp_bookings_ui/wp_rest_api_nonce` or something similar, and retrieve it as such. An example of this can be seen [here][1].

[1]: https://github.com/RebelCode/rcmod-eddbk-rest-api/pull/33/commits/44cc2ae0a7ed4a2d0e52bfced3b97e759fd4257f#diff-1534b3cc8a81650501a35aff7efd144eR621